dpctl/tensor/_elementwise_funcs.py

```diff
@@ -954,10 +964,12 @@
 # U20: ==== LOG         (x)
 _log_docstring = """
 log(x, out=None, order='K')
-Computes the natural logarithm element-wise.
+
+Computes the natural logarithm for each element `x_i` of input array `x`.
```

Comment:
If we used "Test if each element ..." above, we should be consistent and use "Compute the natural logarithm ...". 

No need to change it in this PR, but we should make a note of this and apply changes eventually.